CHANGELOG.md

```diff
@@ -25,11 +22,12 @@ Release date: TBD
 - Added an option to toggle the red dot icon for unread messages (default is on).
 
 #### Windows
+- Update Mattermost icon for desktop notifications in Windows 10.
 - Added an option to toogle the red dot icon for unread messages (default is on).
 
 
 ### Bug Fixes
-
+- Fixed but where window would restore the wrong state under some conditions.
```

Comment:
Will change, but I think it may also address some other restore issues, that are not that obvious.